internal/store/pod.go

```diff
@@ -677,16 +677,21 @@ func createPodInitContainerInfoFamilyGenerator() generator.FamilyGenerator {
 		"",
 		wrapPodFunc(func(p *v1.Pod) *metric.Family {
 			ms := []*metric.Metric{}
-			labelKeys := []string{"container", "image_spec", "image", "image_id", "container_id"}
+			labelKeys := []string{"container", "image_spec", "image", "image_id", "container_id", "type"}
```

Comment:
```suggestion
			labelKeys := []string{"container", "image_spec", "image", "image_id", "container_id", "restart_policy"}
```

let's use the kubernetes specific labels

Reply:
nice but the pod restart policy has a label key name of type, is that okay?
https://github.com/kubernetes/kube-state-metrics/blob/main/internal/store/pod.go#L1191C11-L1191C11

Reply:
yes, that one is in a separate metric, I think if this is in an Info metric it's too generic to call it type.

Reply:
@mrueg 
I understand. From that perspective, restart_policy would be better.